packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -37,18 +37,18 @@ const testKey = "test key";
 const testKey2 = "another test key";
 const testValue = "test value";
 
+const awaitConnected = async (container: IContainer) => {
```

Comment:
nit: ensureContainerConnnected. Similar to the one added here - https://github.com/microsoft/FluidFramework/pull/9272/files.